### PR TITLE
php8 & sylius1.10 compatibility

### PR DESCRIPTION
**Pull request by TonySma**

### CODE REVIEW

**Review by maximehuran — APPROVED** [review]

File: .github/workflows/recipe.yaml [hunk 1]
```diff
@@ -19,8 +19,13 @@ jobs:
...
       matrix:
-        php: [7.4]
-        sylius: [1.8, 1.9]
+        php: ['7.4' ,'8.0']
+        sylius: ["~1.8.0", "~1.9.0", "~1.10.0"]
+        exclude:
+          - php: 8.0
+            sylius: "~1.8.0"
+          - php: 8.0
+            sylius: "~1.9.0"
```
Comment (maximehuran) [inline .github/workflows/recipe.yaml:28]: Do you think it will works with that ?

```
          - php: 8.0
            sylius: ["~1.8.0", "~1.9.0"]
```

Comment (TonySma) [reply .github/workflows/recipe.yaml]: not sure but let's have a try

Comment (maximehuran) [reply .github/workflows/recipe.yaml]: It does not ^^

I lett you revert it and after we will be able to merge \o/

File: .github/workflows/recipe.yaml [hunk 2]
```diff
@@ -19,8 +19,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.4]
-        sylius: [1.8, 1.9]
+        php: ['7.4' ,'8.0']
+        sylius: ["~1.8.0", "~1.9.0", "~1.10.0"]
+        exclude:
+          - php: 8.0
+            sylius: ["~1.8.0"]
```
Comment (maximehuran) [inline .github/workflows/recipe.yaml:26]:
```suggestion
            sylius: "~1.8.0"
```

Comment (TonySma) [reply .github/workflows/recipe.yaml]: my bad, corrected
it's too early, I need a coffee

**Comment by maximehuran** [comment]
Thank you @TonySma <3

**Pull request merged**